test/extended/util/image/image.go

```diff
@@ -19,6 +21,14 @@ var (
 	fromRepository     string
 	images             map[string]string
 
+	releasePullSpecInitializationLock sync.RWMutex
+	releasePullSpecInitialized        bool
+	availablePullSpecs                = map[string]string{
+		"cli":   "image-registry.openshift-image-registry.svc:5000/openshift/cli:latest",
+		"tools": "image-registry.openshift-image-registry.svc:5000/openshift/tools:latest",
```

Comment:
Weird question, what stops us from always using the images coming from the release payload? Do we need to explicitly use the internal registry for that?

Reply:
I don't have the best context for that answer, but I assumed it's to help with most requests staying in cluster. However, I'm not sure how impactful that would be.

I think given that all tests up to now run using that assumption, I figure it's best to err on the side of caution and not change that yet since the majority use case for these changes are the no capabilities periodics.

Reply:
> Weird question, what stops us from always using the images coming from the release payload? Do we need to explicitly use the internal registry for that?

i believe we already *are* always using the images from the release payload.  It's just a question of whether we reference those images:

1) via the local imagestream which is importing from the release payload and results in a pull-through via the internal registry (e..g image-registry.openshift-image-registry.svc:5000/openshift/cli)
or
2) direct reference to the pullspec from the payload  (the same pullspec the imagestream should be importing from)

we can only do (1) when there is an internal registry present, obviously.  The benefit of doing (1) is, theoretically, that the image gets cached in the internal registry and thus optimizes subsequent pulls.  In some cases we might also be using that reference to actually test the imagestream, use of imagestreams, or pullthrough behavior, but i'm not sure.